spec/lib/cyclonedx/report_python_modules_cyclonedx_spec.rb

```diff
@@ -0,0 +1,255 @@
+require_relative '../../spec_helper'
+require 'json'
+
+describe Cyclonedx::ReportPythonModules do
+  describe "#run" do
+    it 'should report all the deps in the unpinned requirements.txt' do
+      repo = Salus::Repo.new('spec/fixtures/python/requirements_unpinned')
+      scanner = Salus::Scanners::ReportPythonModules.new(repository: repo, config: {})
+      scanner.run
+
+      python_cyclonedx = Cyclonedx::ReportPythonModules.new(scanner.report)
+      expected = [
+        {
+          "bom-ref": "pkg:python_requirement/requests",
+          "type": "library",
+          "group": "",
+          "name": "requests",
+          "version": ">=2.5",
+          "purl": "pkg:python_requirement/requests",
```

Comment:
`"purl": "pkg:python_requirement/requests"`

Unsure if "python_requirement" is the desired type here? Simple to change in report_python_modules.rb if necessary.

Reply:
Is `python_requirement` the only possible type?

Reply:
I think the requirements.txt is the only file we're gathering python dependencies from, but that type is currently hardcoded in report_python_modules.rb (https://github.com/coinbase/salus/blob/03279595de981a0aaad278f90f51d797e728e348/lib/salus/scanners/report_python_modules.rb#L20)

Maybe something else would be more suitable than `python_requirement`?